src/main/java/com/example/DoughnutShop.java

```diff
@@ -0,0 +1,20 @@
+package com.example;
+
+public class DoughnutShop {
+
+    public String prepareDoughnut(String doughnutType) {
```

Comment:
Make this an enum

Reply:
If making it an enum, it would be great to make this a `getDescription()` method on the enum itself